common/datavalues/src/arrays/ops/like.rs

```diff
@@ -112,6 +112,7 @@ fn is_like_pattern_escape(c: u8) -> bool {
     c == b'%' || c == b'_' || c == b'\\'
 }
 
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
```suggestion
#[derive(Clone, Debug, PartialEq, Eq, Copy, Hash)]
```

since this is a simple enum.

Reply:
> since this is a simple enum.

got it